### PR TITLE
Replace WOW with AOS

### PR DESCRIPTION
**Pull request by dresl**
I commented line enter_effect_iteration in forms, there is no iteration in AOS.

### CODE REVIEW

File: leonardo/module/web/__init__.py [hunk 1]
```diff
@@ -123,13 +123,13 @@ def extra_context(self):
         }
 
     js_files = [
-        'js/lib/wow.min.js'
+        'js/lib/aos.js'
```
Comment (michaelkuty) [inline leonardo/module/web/__init__.py:126]: maybe minimized version is better

**Comment by michaelkuty** [comment]
Seems ok, thans for doing this

File: leonardo/module/web/templates/base/widget/box.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 
-<div id="{{ widget.fe_identifier }}" class="{{ widget.render_base_classes }} {% if widget.enter_effect_style != 'disabled' %}wow {{ widget.enter_effect_style }}{% endif %}"{% if widget.enter_effect_duration %}data-wow-duration="{{ widget.enter_effect_duration }}s"{% endif %} {% if widget.enter_effect_delay %}data-wow-delay="{{ widget.enter_effect_delay }}s"{% endif %} {% if widget.enter_effect_iteration %}data-wow-iteration="{{ widget.enter_effect_iteration }}"{% endif %} {% if widget.enter_effect_offset %}data-wow-offset="{{ widget.enter_effect_offset }}"{% endif %}>
+<div id="{{ widget.fe_identifier }}" class="{{ widget.render_base_classes }}" {% if widget.enter_effect_style != 'disabled' %}data-aos="{{ widget.enter_effect_style }}"{% endif %}{% if widget.enter_effect_duration %}data-aos-duration="{{ widget.enter_effect_duration }}"{% endif %} {% if widget.enter_effect_delay %}data-aos-delay="{{ widget.enter_effect_delay }}"{% endif %} {% if widget.enter_effect_offset %}data-aos-offset="{{ widget.enter_effect_offset }}"{% endif %} {% comment %}{% if widget.enter_effect_iteration %}data-wow-iteration="{{ widget.enter_effect_iteration }}"{% endif %}{% endcomment %}>
```
Comment (michaelkuty) [inline leonardo/module/web/templates/base/widget/box.html]: instead iteration you could use data-aos-once 0/1 boolean

File: leonardo/module/web/__init__.py (hunk 1 above)
Comment (dresl) [reply leonardo/module/web/__init__.py:126]: https://raw.githubusercontent.com/michalsnik/aos/master/dist/aos.js
This is the file, I don't know why "min" is missing in the name of this file.

Comment (michaelkuty) [reply leonardo/module/web/__init__.py:126]: ok

File: leonardo/module/web/templates/base/widget/box.html (hunk 2 above)
Comment (dresl) [reply leonardo/module/web/templates/base/widget/box.html]: Ok. It is done.

**Pull request merged**